WordPress/Classes/ViewRelated/Suggestions/SuggestionsTableView.swift

```diff
@@ -11,6 +11,15 @@ extension SuggestionType {
 
 @objc public extension SuggestionsTableView {
 
+    // MARK: - API
+
+    /// Returns the a list of prominent suggestions excluding the current user.
+    static func prominentSuggestions(fromPostAuthorId postAuthorId: NSNumber?, commentAuthorsIds: [NSNumber], defaultAccountId: NSNumber?) -> [NSNumber] {
```

Comment:
I've designed the API this way in case we want to cover the UX you suggested in P2 ( pulling all authors above a nested comment ). 

Maybe it's just going to be confusing for now, what do you think?

Reply:
> Maybe it's just going to be confusing for now, what do you think?

I agree, perhaps this should be plural only if / when that feature is implemented. 👍